utils/sms-tool/Makefile

```diff
@@ -0,0 +1,39 @@
+# 2017 - 2021 Cezary Jackiewicz <cezary@eko.one.pl>
+# 2014 lovewilliam <ztong@vt.edu>
+# sms tool for various of 3g modem
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sms-tool
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://github.com/obsy/sms_tool
+PKG_SOURCE_PROTO:=git
+PKG_SOURCE_DATE:=2022-03-21
+PKG_SOURCE_VERSION:=f07699ab5189656f595ffa75872c8938bf4b40ac
+PKG_HASH:=0d114a97eb7aeadd95b49281d29371ba0570382125c34f3f8668d34fd0290f46
+
+PKG_MAINTAINER:=Andrey Butirsky <butirsky@gmail.com>
```

Comment:
@obsy if you wish to take over the maintenance from me, we could change the PKG_MAINTAINER line